booster/tools/rpc-client/RpcClient.hs

```diff
@@ -414,6 +439,10 @@ parseMode =
                     ( RunTarball
                         <$> strArgument (metavar "FILENAME")
                         <*> switch (long "keep-going" <> help "do not stop on unexpected output")
+                        <*> multiString
+                            ( long "run-request"
+                                <> help "Only run the specified request(s), e.g. --run-request \"add-module implies\""
+                            )
```

Comment:
Could maybe use `ApiMethod`  (with a read function)?
And I would suggest the name `--run-only`